### ISSUE TITLE
Suddenly FPS drops while entering/leaving Menus

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#712]
Game runs fine, but sometimes(a lot) the FPS will drastically decrease after entering/leaving the Menu/Inventory and it will stay that way.

---

## Details 
Before the latest patch 1.8.86.0 this bug was not all the time, but after the patch this issue is happing always.

## Game Version
1.8.86.0

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
Just enter or leave the menu/inventory, saddly not everyone has it...
My specs:
GTX 1660 - Driver 546.17
Ryzen 5 3600
16gb RAM
SSD
WINDOWS 11

**Comment by Constellation-VASCO** [comment]
Pretty sure this is the issue with the effects on weapons being recalculated. Try this <https://www.nexusmods.com/starfield/mods/2830>

**Comment by Constellation-VASCO** [comment]
It's highly unlikely this has anything to do with the weapon stuttering bug, especially since it's not actually a stutter, but a constant massive FPS drain. I've experienced it occasionally as well, after leaving a menu, the frame rate will plummet and stay there for an indeterminate amount of time and the dynamic resolution upscaling will go into overdrive to try and compensate. It's unclear what alleviates the issue, sometimes it's just time, sometimes I've fixed it by going in and out of menus repeatedly.
In any case, this is very likely to be out of scope for the Community Patch and require and engine level fix instead.

**Comment by Constellation-VASCO** [comment]
I've had it go both ways.

Sometimes, entering a menu causes massive FPS drops. Other times, it corrects massive FPS drops. Generally, I can eventually get it to go back to normal by entering the main menu or just the tab menu and waiting on it for a couple seconds, and then exiting the menu. If it doesn't fix it, I repeat that cycle until it eventually does, which it always eventually does.

My guess that there is some sort of race condition in the game that is causing non-deterministic behavior, and at least one of the possible outcomes is the FPS drop we see. If so, that's likely to be more difficult to fix, and is more of the kind of issue we should press BGS to fix in an update.
